<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserPopUp</class>
 <widget class="QDialog" name="AddUserPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>338</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>338</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add New User</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/resource.qrc">
    <normaloff>:/icons/cart.png</normaloff>:/icons/cart.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: qlineargradient(
    x1:0, y1:0, x2:0, y2:1,
    stop:0 #FFFFFF,
    stop:1 #E8E8E8
);
}

QLabel {
	color: #34495e;
	font-family: &quot;Segoe UI&quot;;
	font-weight: medium;
    font-size: 12pt;
}


QLineEdit {
	border: none;
	border-bottom: 1px solid #cccccc;

}

QLineEdit:focus {
	border-bottom: 1px solid #34495e;

}


QPushButton {
    background-color: #34495e;
    color: white;
    border: 2px solid #34495e;
    border-radius: 5px;
    font-size: 12pt;
}

QPushButton:hover {
    background-color: #2c3e50;
    border-color: #2c3e50;
}

QPushButton:pressed {
    background-color: #22313f;
    border-color: #22313f;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_26">
        <property name="spacing">
         <number>25</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_28">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="username_label">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Username:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="username_lineEdit_1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_29">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="password_label">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="password_lineEdit_1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_30">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="role_label">
              <property name="text">
               <string>Role:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="role_cb">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Cashier</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Manager</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Admin</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="add_user_btn">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Add User</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resource.qrc"/>
 </resources>
 <connections/>
</ui>
